integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -170,3 +175,51 @@ def view_poi(request, poi_id, region_slug, language_slug):
         raise Http404
 
     return render(request, template_name, {"poi_translation": poi_translation})
+
+
+@json_response
+@require_POST
+@permission_required("cms.view_poi")
+def auto_complete_address(request):
+    """
+    Autocomplete location address and coordinates
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :raises ~django.http.Http404: If no location was found for the given address
+
+    :return: The address and coordinates of the location
+    :rtype: ~django.http.JsonResponse
+    """
+    data = json.loads(request.body.decode("utf-8"))
+
+    if not settings.NOMINATIM_API_ENABLED:
+        return HttpResponse(_("Location service is disabled"), status_code=503)
+
+    street_input = data.get("street")
+    postcode_input = data.get("postcode")
+    city_input = data.get("city")
+
+    nominatim_api_client = NominatimApiClient()
+
+    result = nominatim_api_client.search(
+        street=street_input,
+        postalcode=postcode_input,
+        city=city_input,
+        addressdetails=True,
+    )
```

Comment:
> From those coordinates the city will be calculated. This is needed because data which geocode calclulates with the combination of street and postcode has the address as one string without attribute like "city" or "comunity (Landkreis)" and there is no guaranty on which positioin the city name stands: whether the address is in a Landkreis or kreisfreie Stadt makes a difference on the position of city name.

Instead of performing two searches - first for the coordinates and then the reverse search for the address - we can also combine them by setting `addressdetails=True` (see [docs](https://geopy.readthedocs.io/en/stable/#nominatim)).
Then the address is given as dictionary and not as string.
Also, instead of importing Nominatim directly in this view, I think it's better to adapt our Nominatim api client to our needs and then call the changed methods of the client.

Reply:
Oh, I overlooked it 😅 it is what I wanted to have!